### ISSUE TITLE
Bind to ip not working correctly

### ISSUE THREAD

**Issue by sulaweyo** [gerbera/gerbera#909]
OS: Arch
version 1.5.0

I have a dhcp assigned IP (192.168.1.5) and an additional IP (192.168.1.4) configured on my primary NIC and i have defined the IP option to 192.168.1.5 in the config. The second IP is for a different DNS name i do not want to use for gerbera. On startup gerbera logs the correct IP:
```
Jun 12 13:09:42 eos gerbera[1932178]: 2020-06-12 13:09:42   info: Initialized port: 50500
Jun 12 13:09:42 eos gerbera[1932178]: 2020-06-12 13:09:42   info: Server bound to: 192.168.1.5
Jun 12 13:09:42 eos gerbera[1932178]: 2020-06-12 13:09:42   info: The Web UI can be reached by following this link: http://192.168.1.5:50500/
```

What it actually does is bind the virtual IP 192.168.1.4 that is assigned to the same interface:
`tcp        0      0 192.168.1.4:50500       0.0.0.0:*               LISTEN      1932178/gerbera `

As a result i can access the web interface if i use 192.168.1.4 but not on the IP i defined in the config. 

From what i see in the logs when i tested various configurations it looks like gerbera uses that IP only to identify the interface that it should bind but then for binding actually uses the first IP it sees on the interface. When i define the interface instead of the IP i'll get a bind for 192.168.1.4 as well. I tried to understand the code and to me it looks like this is done by libupnp, not sure if it can actually be set but then the log is still wrong and the option does not do what it says in the documentation

**Comment by whyman** [comment]
Hello, thanks for taking the time to report.

We work out which interface the IP is on, if you provide and IP and then simply pass that to libupnp's `UpnpInit2`. Obviously at that point the IP is lost.

As far as I know, there is no way to pass an IP instead...

**Comment by sulaweyo** [comment]
Thought so. In this case it would be cool if the docu could be adjusted as the bind option does not actually bind that IP. That's what i would have expected if there is the option to bind an IP

**Comment by whyman** [comment]
Yeah, its odd behaviour! Ideally what we need is the ability to pass the IP to libupnp directly

**Comment by limburgher** [comment]
Seeing this on gerbera 1.12.1 with libupnp 1.14.7. In my case the second IP comes from keepalived.  Is there any movement on this either here or in libupnp?

**Comment by KarlStraussberger** [comment]
Sorry, haven't seen any change there. You may try libnpupnp.

**Comment by bkuhls** [comment]
> Sorry, haven't seen any change there. You may try libnpupnp.

Thanks for the hint, this problem was solved here by moving from libupnp 1.14.25 to libnpupnp 6.3.0.
